README.md

```diff
@@ -34,7 +35,50 @@ var User = Arrow.Model.extend('user', {
 			attachment_content: { name: 'content', type: String }
 		},
 		connector: 'appc.mongo'
-	});
+	})
+```
+
+#### Selecting Fields
+
+To select precise fields from a data source, for the majority of cases making a model with fields with keys which match those of a specified model, and specifying the correct type is enough:
```

Comment:
Got precise from the existing verbiage, can change it though. 
While it fits with our understanding of reduction, the use of that word is a bit confusing in this case, especially when it's using Arrow.Model.Extend.

"Composite model creation allows you to expose a subset of fields from a source model...."
- The key to this paragraph though is (*the majority of cases*). 

"... In the majority of cases, if the name of the field in the composite model matches the name in the source model, then specifying the source model and correct type is enough."

... however ... (next paragraph)